### DISCUSSION TITLE
Can't run git clone without sudo

### DISCUSSION THREAD

**Discussion by g1sbi** [abraunegg/onedrive#1850] [Q&A] [answered]
I'm on Pop 21.10, I followed all the dependencies installations but I can't run the git clone command without sudo. How do I fix this? Sorry for the omega-noob question but I'm new to Linux. Thanks :)

**Comment by abraunegg** [comment, marked as answer]
@Gisbitus 
Running `git clone .... ` as a standard user should be possible - I do not know *why* you cant run this .. 'git' is a 'user-space' binary and does not need 'root privileges' to run .. and on my test system - I can run 'git' without sudo - which is 100% normal:

```
Welcome to Pop!_OS 21.10 (GNU/Linux 5.15.8-76051508-generic x86_64)

 * Homepage: https://pop.system76.com
 * Support:  https://support.system76.com

Last login: Mon Feb 21 07:01:16 2022 from 172.16.16.212
alex@pop-os:~$ git clone https://github.com/abraunegg/onedrive.git
Cloning into 'onedrive'...
remote: Enumerating objects: 5423, done.
remote: Counting objects: 100% (758/758), done.
remote: Compressing objects: 100% (390/390), done.
remote: Total 5423 (delta 491), reused 572 (delta 349), pack-reused 4665
Receiving objects: 100% (5423/5423), 3.82 MiB | 4.30 MiB/s, done.
Resolving deltas: 100% (3735/3735), done.
alex@pop-os:~$ 
```
